utils/_context/_scenarios/integrations.py

```diff
@@ -114,8 +122,6 @@ def __init__(
 
     def configure(self, config):
         super().configure(config)
-        if not self.replay:
```

Comment:
Why removing this ?

Reply:
We don't need to check for AWS env variables since we are no longer communicating with actual AWS